src/lib/rest/RestService.cpp

```diff
@@ -759,7 +759,7 @@ static std::string restService(ConnectionInfo* ciP, RestService* serviceV)
   std::string  details = std::string("service '") + ciP->url + "' not recognized";
   std::string  answer;
 
-  restErrorReplyGet(ciP, SccBadRequest, "service not found", &answer);
+  restErrorReplyGet(ciP, SccBadRequest, "service not found. Check your URL as probably it is wrong.", &answer);
```

Comment:
Looking to other "two sentences error descriptions", such as `More than one matching entity. Please refine your query` I think the following should be better:

```suggestion
  restErrorReplyGet(ciP, SccBadRequest, "Service not found. Check your URL as probably it is wrong.", &answer);
```

I understand the change is easy (just a "replace all" in files). Note that no changes in Content-Lenght is needed in .test files, as the the message still having the same number of characters.